airbyte-api/src/main/openapi/config.yaml

```diff
@@ -453,6 +456,7 @@ paths:
           application/json:
             schema:
               $ref: "#/components/schemas/CustomSourceDefinitionUpdate"
+        required: true
```

Comment:
Thanks for your contribution. In https://github.com/airbytehq/airbyte/issues/19670 we are moving away from this function, and will use updateSourceDefinition to replace this.

A tl;dr version of the linked ticket is:

1) We are moving away updateCustomSourceDefinition and going to use updateSourceDefinition (same for destination)
2) We are moving away deleteCustomSourceDefinition and going to use deleteSourceDefinition (same for destination)
3) We are moving away from SourceDefinitionCreate and going to use CustomSourceDefinitionCreate (same for destination). (Note in this bullet point we are actually going to use `custom` in API because created definitions are always going to be custom connectors, while in the last 2 we are moving away from `custom` so both custom connector or public connector share the same path)